src/pages/Library/style.css

```diff
@@ -1,14 +1,24 @@
 .App-library {
+    overflow-y: auto;
 }
 
-.libraryCard{
+.App-library-book-list {
```

Comment:
can commented out style be removed if not required

Reply:
This page needs styling, I'd rather leave it in there in case it needs to be put back